code/game/objects/items/implants/implant_misc.dm

```diff
@@ -34,10 +34,15 @@
 	. = ..()
 	uses--
 	to_chat(imp_in, "<span class='notice'>You feel a sudden surge of energy!</span>")
+	imp_in.SetSleeping(0)
 	imp_in.SetStun(0)
 	imp_in.SetKnockdown(0)
 	imp_in.SetUnconscious(0)
-	imp_in.adjustStaminaLoss(-75)
+	imp_in.adjustStaminaLoss(-200)
+	imp_in.updatehealth()
+	imp_in.update_stamina()
```

Comment:
adjuststaminaloss() already goes through update_stamina() and updatehealth()

Reply:
My main problem previously was that _something_ didn't update properly in the same exact tick and even after `adjustStaminaLoss()`, `lying = 0` with `update_canmove()` would make you drop right back to the floor.
I suppose I just kept adding stuff until it worked and didn't bother to remove it afterwards.
Let's see how it works with the edits.

Reply:
After a lot of testing, I'm actually ready to defend the existence of those procs in the PR.
Without them, stamina updates the tick after, rather than immediately.